pkg/dns-server/resolver/server.go

```diff
@@ -0,0 +1,211 @@
+package resolver
+
+import (
+	"sync"
+
+	"github.com/pkg/errors"
+	"go.uber.org/multierr"
+
+	"github.com/miekg/dns"
+
+	"github.com/Kong/kuma/pkg/core"
+)
+
+var (
+	simpleDNSLog = core.Log.WithName("dns-server-resolver")
+)
+
+type (
+	VIPList           map[string]string
```

Comment:
What 'V' stands for?

Reply:
VIP is for Virtual IP. That is pretty common acronym IMO>

Reply:
I don't mind, I just really didn't know :)